src/model/InteractionModel.ts

```diff
@@ -16,9 +16,18 @@ export class InteractionModel implements IModel {
 
     // Parse the all-in-one interaction model as a file
     public static fromFile(interactionModelFile: any): InteractionModel {
-        const data = fs.readFileSync(interactionModelFile);
-        const json = JSON.parse(data.toString());
-        return InteractionModel.fromJSON(json);
+        try {
+            const data = fs.readFileSync(interactionModelFile);
+            const json = JSON.parse(data.toString());
+            return InteractionModel.fromJSON(json);
+        } catch (error) {
```

Comment:
is there a specific error for this case?

Reply:
the error happens in  fs.readFileSync when the file is not found, but there are other types of errors, for permission issues or incorrect JSON that we still want to be thrown